trackpy/plots.py

```diff
@@ -260,6 +260,7 @@ def pairwise(iterable):
         _plot_style.update(markeredgecolor=color[-1])
         ax.plot(centroids['x'][high], centroids['y'][high],
                 **_plot_style)
+    ax.invert_yaxis()
```

Comment:
I just wanted to be sure that it is inverted, don't know what is happening in between the `if ... then` clause below line 233. But if you're sure that it's safe I will change it.